src/cli.c

```diff
@@ -924,6 +924,11 @@ cli_build_asset(char *fpath, struct dirent *dp)
 	bopt = cli_buildopt_default();
 	name = cli_strdup(dp->d_name);
 
+	/* Ignore hidden files and some editor files */
```

Comment:
Probably best to move this higher up and use the d_name member from the dirent data structure so we don't end up copying d_name into name and then not using it.

Reply:
Yeah. Nice catch.